website/content/v1.6/talos-guides/install/boot-assets.md

```diff
@@ -170,27 +170,44 @@ The base profile can be customized with the additional flags to the imager:
   For example `-console` removes all `console=<value>` arguments, whereas `-console=tty0` removes the `console=tty0` default argument.
 * `--system-extension-image` allows to install a system extension into the image
 
+### Extension Image Reference
+
+While Image Factory automatically resolves the extension name into a matching container image for a specific version of Talos, `imager` requires the full explicit container image reference.
+The `imager` also allows to install custom extensions which are not part of the official Talos Linux system extensions.
+
+To get the official Talos Linux system extension container image reference matching a Talos release, use the [following command](https://github.com/siderolabs/extensions?tab=readme-ov-file#installing-extensions):
+
+```shell
+crane export ghcr.io/siderolabs/extensions:{{< release >}} | tar x -O image-digests | grep EXTENSION-NAME
+```
+
+> Note: this command is using [crane](https://github.com/google/go-containerregistry/blob/main/cmd/crane/README.md) tool, but any other tool which allows
+> to export the image contents can be used.
+
+For each Talos release, the `ghcr.io/siderolabs/extensions:VERSION` image contains a pinned reference to each system extension container image.
+
 ### Example: Bare-metal with Imager
 
 Let's assume we want to boot Talos on a bare-metal machine with Intel CPU and add a `gvisor` container runtime to the image.
 Also we want to disable predictable network interface names with `net.ifnames=0` kernel argument and replace the Talos default `console` arguments and add a custom `console` arg.
 
 First, let's lookup extension images for Intel CPU microcode updates and `gvisor` container runtime in the [extensions repository](https://github.com/siderolabs/extensions):
 
-```text
-ghcr.io/siderolabs/gvisor:20231214.0-v1.5.0-beta.0
-ghcr.io/siderolabs/intel-ucode:20230613
+```shell
+$ crane export ghcr.io/siderolabs/extensions:{{< release >}} | tar x -O image-digests | egrep 'gvisor|intel-ucode'
```

Comment:
modified the example to show full hashes

Reply:
`egrep` is kinda deprecated, `grep -E` please